Formula/aws-sdk-cpp.rb

```diff
@@ -0,0 +1,44 @@
+class AwsSdkCpp < Formula
+  desc "AWS SDK for C++"
+  homepage "https://github.com/aws/aws-sdk-cpp"
+  url "https://github.com/aws/aws-sdk-cpp/archive/0.10.6.tar.gz"
+  sha256 "01ad50677e2bde3bcd51128a9e043f76371e9642a5bea235637ce8e450eec5de"
+  head "https://github.com/aws/aws-sdk-cpp.git"
```

Comment:
Probably because that added blank line happens on the following line, line 7. To get comments to be auto-hidden, they should probably be placed at the insertion point instead of the line preceding it.